pkg/kubelet/prober/prober.go

```diff
@@ -98,7 +98,7 @@ func (pb *prober) probe(ctx context.Context, probeType probeType, pod *v1.Pod, s
 	}
 
 	result, output, err := pb.runProbeWithRetries(ctx, probeType, probeSpec, pod, status, container, containerID, maxProbeRetries)
-	if err != nil || (result != probe.Success && result != probe.Warning) {
+	if err != nil || (result != probe.Success && result != probe.Warning) && result != probe.Unknown {
```

Comment:
Would it simplify the code if we check for `probe.Success`, `probe.Failure`, `probe.Warning` and `probe.Unknown` separately?